untitled/src/homework2/task2.java

```diff
@@ -0,0 +1,17 @@
+package homework2;
+
+import java.util.Scanner;
+
+public class task2 {
+    public static void main(String[] args) {
```

Comment:
Not wrong, but if you know th enumber of iteraction will be easy to use for loop
        for(int i = 1 ; i < 1001 ; i++) {
            System.out.println(i);
        }